chatwoot/chatwoot
# chore: Suppress the unnecessary CSRF warning

- suppress the unnecessary CSRF warnings
- update codespaces to handle webpack dev server

# Testing 
- deployed on the staging server
- load the widget, see the logs for the post endpoints like toggle typing 
- make sure the warning is not shown anymore.
